lib/gems/pending/appliance_console/external_httpd_configuration.rb

```diff
@@ -106,6 +106,9 @@ def configure_sssd_domain(config, domain)
           pattern = "[domain/#{Regexp.escape(domain)}].*(\n)"
           config[/#{pattern}/, 1] = "\nldap_user_extra_attrs = #{ldap_user_extra_attrs}\n"
         end
+
+        pattern = "ldap_user_extra_attrs.*(\n)"
+        config[/#{pattern}/, 1] = "\nentry_cache_timeout = 600\n"
```

Comment:
@abellotti I had thought of that but I realized that since we just added the ldap_user_extra_attrs line to the domain section there seems to be no chance that the it will be in a different section.